src/schema/rules/sidecars/asl.yaml

```
@@ -16,20 +16,22 @@ MRIASLTextOnly:
         MagneticFieldStrength: required
         MRAcquisitionType: required
         EchoTime: required
-        SliceTiming:
-            level: recommended
-            level_addendum: required if `MRAcquisitionType` is defined as `2D`
         RepetitionTimePreparation: required
-        FlipAngle:
-            level: recommended
-            level_addendum: required if `LookLocker` is `true`
 
-MRIASLTextOnlySliceTiming:
+MRIASLTextOnlySliceTimingRec:
+    selectors:
+    - modality == "MRI"
+    - datatype == "perf"
+    - suffix == "asl"
+    - sidecar.MRAcquisitionType == "2D"
+    fields:
+        SliceTiming: recommended
```

```suggestion
        SliceTiming: required

MRIASLTextOnlySliceTimingRec:
     selectors:
     - modality == "MRI"
     - datatype == "perf"
     - suffix == "asl"
     - sidecar.MRAcquisitionType != "2D"
     fields:
         SliceTiming: recommended
```

Is this what you mean by atomic rules?

I think this will make rendering tables harder, since the original rules that have level addenda can't be place in the original tables because the selectors are different. I think we'll also need tests that check that all rules are mutually exclusive, or reviewers will have trouble identifying any flaws in contributors' logic.